### DISCUSSION TITLE
Ways to optimize image for VNC?

### DISCUSSION THREAD

**Discussion by kevinmnm** [jlesage/docker-firefox#192] [Q&A]
Hello, I'm using the [repo](https://github.com/jlesage/docker-firefox) to build my own custom image for VNC deployment. I noticed the performance isn't great when it comes to viewing maps or dealing with heavy data. Thus, I wanted to find a way to optimize (smallest possible image size and more performant user experience) as much as possible and need some help as I'm not so familiar with all the packages listed in the [Dockerfile](https://github.com/jlesage/docker-firefox/blob/master/Dockerfile). Any suggestion on how I can approach this will be greatly appreciated.

**About my app:**
- Will always enforce `FF_KIOSK=1` and preset `FF_OPEN_URL=`, making client limited to only ONE webpage and the webpage only.
- No need for feature such as "downloads" or access to filesystem.
- Performance is top priority (I wonder `APP_NICENESS=19` will boost performance?).
- Container running VNC is less likely delete, meaning it will stack up over time.

**Comment by jlesage** [comment]
> I noticed the performance isn't great when it comes to viewing maps or dealing with heavy data.

What symptoms do you experiment exactly ?
